app/views/RegisterView.js

```diff
@@ -130,6 +130,8 @@ class RegisterView extends React.Component {
 			Alert.alert(I18n.t('Oops'), e.data.error);
 		}
 		this.setState({ saving: false });
+		const { navigation } = this.props;
```

Comment:
On both cases you can't redirect to RoomsList if some error occurred on `catch`. 😁
I think it's better you can put it after `await loginRequest({ user: email, password });`.

Reply:
So basically I have to put ```await loginRequest({ user: email, password });``` before ```const { navigation } = this.props``` , is it @djorkaeffalexandre  ?